docs/install/setup-storybook.md

```diff
@@ -40,9 +40,9 @@ An example of a final `main.js` would look something like the following:
 const path = require('path');
 
 module.exports = {
-  "stories": [
-    "../src/**/*.stories.mdx",
-    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+  "Stories": [
```

Comment:
I think this might need to be lowercased :)

Reply:
Done